app/views/container_dashboard/show.html

```diff
@@ -0,0 +1,36 @@
+<div ng-app="containerDashboard" ng-controller="containerDashboardController as dashboard" class="container-fluid container-tiles-pf containers-dashboard miq-dashboard-view">
+    <div class="row row-tile-pf" >
+        <div class="col-xs-6 col-sm-4 col-md-2">
+            <div pf-aggregate-status-card status="objectStatus.providers" show-top-border="true" layout="tall" class="card-pf-title-container-dash"></div>
+        </div>
+        <div class="col-xs-12 col-sm-12 col-md-10">
+            <div class="row">
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.nodes" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.containers" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.registries" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.projects" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash" ></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.pods" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.services" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.images" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
```

Comment:
can "col-xs-6 col-sm-6 col-md-3" simplified to a shorter name to avoid copy pasting of long names?